moveit2/README.md

```diff
@@ -105,56 +105,4 @@ You can also test the graphics output by running 'glmark2'. A window like the fo
 
 NOTE: The *xauth add* command will have to be run each time a MoveIt2 container is started.
 
-Once GUIs and graphics are verified to be working from the container, you can move on to the next step, running the MoveIt2 MoveGroup demo.
-
-## Running Moveit2's MoveGroup Demo
-
-From a container that has been configured and verified to run GUI and graphics apps, run the following command:
-
-```
-ros2 launch run_move_group run_move_group.launch.py
-```
-
-You will see a lot of console output and the rviz2 window will appear:
-
-![rviz2 window](resources/moveit2-rviz.png)
-
-Now, from a second host terminal, either connect to the existing container, using 'docker container list' and 'docker exec', similar to the following:
-
-```
-$ docker container list
-CONTAINER ID   IMAGE                  COMMAND                  CREATED          STATUS          PORTS     NAMES
-5c08ddc48d79   openrobotics/moveit2   \"/moveit2_entrypoint…"   15 minutes ago   Up 15 minutes             angry_ptolemy
-```
-```
-$ docker exec -it 5c08ddc48d79 /bin/bash /moveit2_entrypoint.sh
-```
-
-or launch and configure a new container per the above instructions for configuring a running container for GUI applications.
-
-In either case, from a container that has been configured and verified to run GUI and graphics apps, you can now run the second part of the demo:
-
-```
-ros2 launch run_move_group run_move_group_interface.launch.py
-```
-This brings up an xterm, like this:
-
-![xterm window](resources/moveit2-xterm.png)
-
-You can now press ENTER multiple times to step through the demo.
-
-## Running Unit Tests for the MoveGroup Demo
-
-To run the unit tests for the MoveGroup demo, run the following command from the container command line at the top of the MoveIt2 workspace (/root/src/moveit2_ws/):
-
-```
-colcon test --packages-select moveit_ros_move_group
-```
-
-The test results will be in the test output directory, /root/src/moveit2_ws/build/moveit_ros_move_group/test_results/.
-
-## Open Issues
-
-* With the latest MoveIt2 source, compiled against the latest Space ROS source code, there is currently a compile-time error in realtime_server_goal_handle.h. There is a patch installed (patch-1.diff) in the Dockerfile to work around this.
-
-* There are two versions of libcurl installed on the system, one installed along with the curl utility (installed in the base Space ROS image) and another that comes along with other installed libraries. There are warnings output when running utilities, such as cmake, that load libcurl. However, this seems not to affect the build process.
+Once GUIs and graphics are verified to be working from the container, you can move on to the next step: [building and running MoveIt2 Tutorials](../moveit2_tutorials/README.md).
```

Comment:
Was dropping all the content after this initial sentence intentional?

Reply:
@nuclearsandwich  Yes, the maintainers of the MoveIt2 repository have moved out some of the sample apps into the moveit2_tutorials repo. In particular, the move_group demo that I was using before has been eliminated. So, this section is now obsolete. In the moveit2_tutorials README, I've added a different test to run and have pointed to the online documentation for the details of running the test.

Reply:
Thanks for elaborating!